extras/install_libboost.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Install the boost library
+wget --no-verbose https://dl.bintray.com/boostorg/release/1.72.0/source/boost_1_72_0.tar.gz
+tar -xzf boost_1_72_0.tar.gz
+rm -fr boost_1_72_0.tar.gz
+cp -fr boost_1_72_0 $HOME/.cache
```

Comment:
Should it be installed to a directory `$HOME/.cache/boost.install` to make it consistent with the other libraries?

Reply:
@hakonsbm Yes, good point!

Reply:
I have applied the change you suggested.
Seems that GitHub/Travis has problems to retrieve the changed files when I create a PR against your repo. The build ran successfully under my account on Travis: [Travis build](https://travis-ci.org/github/gtrensch/nest-simulator/builds/677304883)
I think it's safe if you merge the PR.

Reply:
Yes, looks like the static check doesn't handle PRs that are not against master that well.